app/models/pull_request.rb

```diff
@@ -13,7 +14,7 @@ class PullRequest < ApplicationRecord
 
   scope :within_month_limit, -> do
     where('pull_requests.gh_updated_at > ?',
-      Time.current - ENV['PULL_REQUEST_MONTH_LIMIT'].to_i.months)
+          Time.current - ENV['PULL_REQUEST_MONTH_LIMIT'].to_i.months)
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.